redi-connect-front/src/pages/front/signup/SignUp.tsx

```diff
@@ -59,16 +60,17 @@ export const validationSchema = Yup.object({
   })
 })
 
-export type SignUpPageType = {
+type SignUpPageType = {
   type: | 'mentor' | 'mentee'
 }
 
-export type SignUpUserType =
-| 'public-sign-up-mentee-pending-review'
-| 'public-sign-up-mentor-pending-review'
+type SignUpUserType = Extends<
```

Comment:
This is why I don't about typescript. Makes the code hard to understand for most of the ppl.